shell/common/shell.cc

```diff
@@ -475,12 +475,12 @@ Shell::~Shell() {
 }
 
 std::unique_ptr<Shell> Shell::Spawn(
-    Settings settings,
+    RunConfiguration run_configuration,
```

Comment:
This was changed from Settings to RunConfiguration since the spawning mechanism can't consume most of the Settings data anyway (since we're spawing off of a previous Shell which was already built using existing Settings data).